standard/tier/wikis/formula1/tier_data.lua

```diff
@@ -56,9 +56,17 @@ return {
 			link = 'Karting Tournaments',
 			category = 'Karting Tournaments',
 		},
+		{
+			value = '7',
+			sort= 'A10',
```

Comment:
- space between `sort=`
- A10 is a bad idea since it is using string sort, so A10 comes before A2